src/main/java/com/matchmate/tagsmanagementservice/common/mappers/RequestTagMapper.java

```diff
@@ -26,7 +26,9 @@ public static RequestTagDocument toDocumentWithoutRequestAt(RequestTag requestTa
         return new RequestTagDocument(
                 requestTag.getId().fromValue(),
                 requestTag.getName(),
-                requestTag.getRequests());
+                requestTag.getRequests(),
+                OffsetDateTime.now()
```

Comment:
nao eh legal inserir valor assim no mapper, a ideia eh que o mapper seja apenas um parse da dados

Reply:
vc pode criar um construtor especifico para esse caso, ou pode criar um factory

Reply:
tava pensando em reusar o toDocument e passar so o offsernow como parametro, é uma boa?
![image](https://github.com/match-mate/tags-management-service/assets/67878286/644b39ec-4b7e-4412-8edd-00d670afca28)

Reply:
Pode ser, esse é o caso de primeira solicitação né? Se for, coloca isso em um método privado que deixa explícito que é o caso de primeira solicitação da request

Reply:
Ah, eu mudei a abordagem desse código, depois da uma olhada